### ISSUE TITLE
Compile errors on CI

### ISSUE THREAD

**Issue by ryanisaacg** [koute/stdweb#309]
I'm getting this compile error on CI:
```
error: custom attribute panicked
  --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/stdweb-0.4.11/src/webcore/macros.rs:32:9
   |
32 |         #[$crate::private::js_raw_attr]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   | 
  ::: /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/stdweb-0.4.11/src/webcore/initialization.rs:20:5
   |
20 |     stdweb_internal_runtime_initialize!( __js_raw_asm );
   |     ---------------------------------------------------- in this macro invocation
   |
   = help: message: called `Option::unwrap()` on a `None` value
```

**Comment by koute** [comment]
Do you have a link to a job or a reproduction crate?

**Comment by koute** [comment]
Released `stdweb` 0.4.12 with that `unwrap` replaced with an `expect`. Now it should be obvious why this happens.

**Comment by ryanisaacg** [comment]
Does cargo-web support the `check` subcommand?

**Comment by koute** [comment]
It does, but it's not yet released.

**Comment by ryanisaacg** [comment]
Also, [here](https://github.com/ryanisaacg/quicksilver/pull/414) is the build that is trying to bump the version.

**Comment by koute** [comment]
Newly released `cargo-web` 0.6.23 should support `cargo web check` subcommand now.
